graphql_compiler/__init__.py

```diff
@@ -218,3 +222,69 @@ def graphql_to_redisgraph_cypher(schema, graphql_query, parameters, type_equival
         schema, graphql_query, type_equivalence_hints=type_equivalence_hints)
     return compilation_result._replace(
         query=insert_arguments_into_query(compilation_result, parameters))
+
+
+def get_sqlalchemy_schema_info(
+    vertex_name_to_table, direct_edges, junction_table_edges, dialect,
+    class_to_field_type_overrides=None
+):
+    """Return a SQLAlchemyInfo from the metadata.
+
+    Args:
+        vertex_name_to_table: dict, str -> SQLAlchemy Table. This dictionary is used to generate the
```

Comment:
I use the variable name `vertex_name_to_table` instead of `tables` since there are other `tables`, (junction tables), that are not included in this dict.